sql_metadb/derived_tables/finance_po_inv_transactions.sql

```diff
@@ -0,0 +1,266 @@
+        -- organizations
+        invoice_organizations.id AS invoice_vendor_id,
+        jsonb_extract_path_text(invoice_organizations.jsonb, 'name') AS invoice_vendor_name
+    FROM
+        folio_orders.po_line AS po_line
+        LEFT JOIN folio_orders.purchase_order AS purchase_order ON purchase_order.id = po_line.purchaseorderid
+        LEFT JOIN folio_invoice.invoice_lines AS invoice_lines ON jsonb_extract_path_text(invoice_lines.jsonb, 'poLineId')::uuid = po_line.id
+        LEFT JOIN folio_invoice.invoices AS invoices ON invoices.id = invoice_lines.invoiceid
+        LEFT JOIN folio_organizations.organizations AS invoice_organizations ON jsonb_extract_path_text(invoices.jsonb, 'vendorId')::uuid = invoice_organizations.id
+),
+finance AS (
+    SELECT
+        fiscal_year.id AS fiscal_year_id,
+        jsonb_extract_path_text(fiscal_year.jsonb, 'code') AS fiscal_year,
+        ledger.id AS ledger_id,
+        jsonb_extract_path_text(ledger.jsonb, 'name') AS ledger_name,
+        budget.id AS budget_id,
+        jsonb_extract_path_text(budget.jsonb, 'name') AS budget_name,
+        fund.id AS fund_id,
+        jsonb_extract_path_text(fund.jsonb, 'code') AS fund_code
+    FROM
+        folio_finance.fiscal_year AS fiscal_year
+        LEFT JOIN folio_finance.ledger AS ledger ON ledger.fiscalyearoneid = fiscal_year.id
+        LEFT JOIN folio_finance.fund AS fund ON fund.ledgerid = ledger.id
+        LEFT JOIN folio_finance.budget AS budget ON budget.fundid = fund.id
+            AND budget.fiscalyearid = fiscal_year.id
+),
+transactions AS (
+    SELECT
+        transaction.id AS transaction_id,
+        jsonb_extract_path_text(transaction.jsonb, 'fromFundId')::uuid AS fromFundId,
+        jsonb_extract_path_text(transaction.jsonb, 'amount')::numeric(19, 4) AS transaction_amount,
+        jsonb_extract_path_text(transaction.jsonb, 'currency') AS transaction_currency,
+        jsonb_extract_path_text(transaction.jsonb, 'sourceInvoiceLineId')::uuid AS sourceInvoiceLineId,
+        jsonb_extract_path_text(transaction.jsonb, 'encumbrance', 'sourcePurchaseOrderId')::uuid AS sourcePurchaseOrderId,
+        jsonb_extract_path_text(transaction.jsonb, 'encumbrance', 'sourcePoLineId')::uuid AS sourcePoLineId,
+        jsonb_extract_path_text(transaction.jsonb, 'encumbrance', 'initialAmountEncumbered')::numeric(19, 4) AS transaction_encumbrance_initial_amount,
+        jsonb_extract_path_text(transaction.jsonb, 'encumbrance', 'amountExpended')::numeric(19, 4) AS transaction_encumbrance_amount_expended,
+        jsonb_extract_path_text(transaction.jsonb, 'encumbrance', 'orderType') AS transaction_encumbrance_order_type,
+        jsonb_extract_path_text(transaction.jsonb, 'encumbrance', 'subscription')::boolean AS transaction_encumbrance_subscription
+    FROM
+        folio_finance.transaction AS TRANSACTION
```

Comment:
This identifier is in all-caps; identifiers should be lowercase only.  Since this one is also unnecessary, I will remove it.